--- psplash-44afb7506d43cca15582b4c5b90ba5580344d75d/psplash-config.h	2021-10-29 14:44:47.000000000 +0200
+++ psplash-patched/psplash-config.h	2023-09-27 20:57:11.242033341 +0200
@@ -17,14 +17,7 @@
 #endif
 
 /* Bool indicating if the image is fullscreen, as opposed to split screen */
-#ifndef PSPLASH_IMG_FULLSCREEN
-#define PSPLASH_IMG_FULLSCREEN 0
-#endif
-
-/* Bool indicated if the progress bar should be disabled */
-#ifndef PSPLASH_DISABLE_PROGRESS_BAR
-#define PSPLASH_SHOW_PROGRESS_BAR 1
-#endif
+#define PSPLASH_IMG_FULLSCREEN 1
 
 /* Position of the image split from top edge, numerator of fraction */
 #define PSPLASH_IMG_SPLIT_NUMERATOR 5
